src/hotspot/share/runtime/deoptimization.hpp

```diff
@@ -44,7 +44,8 @@ class Deoptimization : AllStatic {
   friend class VMStructs;
   friend class EscapeBarrier;
 
- public:
```

Comment:
You can revert changes to this file since this wasn't part of your change.  Somebody can fix this some other time when they're changing this file.